tests/console/autofs.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Mount an iso file using autofs/automount
+#   
+# Maintainer: Antonio Caristia <acaristia@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    my $autofs_conf_file = '/etc/auto.master';
+    my $autofs_map_file = '/etc/auto.master.d/autofs_regression_test.autofs';
+    my $test_conf_file = '/etc/auto.iso';
+    my $test_mount_dir = '/mnt/test_autofs_local';
+    my $file_to_mount = '/tmp/test-iso.iso';
+    my $test_conf_file_content = "echo  iso     -fstype=auto,ro         :$file_to_mount > $test_conf_file";
+    assert_script_run("mkdir -p $test_mount_dir");
+    assert_script_run("if=/dev/urandom of=$test_mount_dir/README bs=4024 count=1");
```

Comment:
aren't you missing a `dd` or something here?

Reply:
thanks :) fixed!